### ISSUE TITLE
Execution error in higher version of NodeJS

### ISSUE THREAD

**Issue by byongshintv** [zetaraku/arcade-songs#9]
The installation manual says to use LTS or a higher NodeJS version, but in reality it is impossible to run with the error below.
```shell
PS C:\Users\*\Desktop\arcade-songs> yarn dev
yarn run v1.22.19
$ nuxt
i Parsed 1 files in 0.4 seconds                                                                 @nuxt/content 19:26:02
i Sentry reporting is disabled (no DSN has been provided)                                         nuxt:sentry 19:26:02

   ╭───────────────────────────────────────╮
   │                                       │
   │   Nuxt @ v2.15.8                      │
   │                                       │
   │   ▸ Environment: development          │
   │   ▸ Rendering:   server-side          │
   │   ▸ Target:      server               │
   │                                       │
   │   Listening: http://localhost:3000/   │
   │                                       │
   ╰───────────────────────────────────────╯

i Preparing project for development                                                                           19:26:03
i Initial build may take a while                                                                              19:26:03
i Discovered Components: .nuxt/components/readme.md                                                           19:26:03
√ Builder initialized                                                                                         19:26:03
√ Nuxt files generated                                                                                        19:26:03

 WARN  Browserslist: caniuse-lite is outdated. Please run:                                                    19:26:04
  npx update-browserslist-db@latest
  Why you should do it regularly: https://github.com/browserslist/update-db#readme


* Client █████████████████████████ compiling (0%)


* Server █████████████████████████ building (10%) 1/1 modules 0 active


node:internal/crypto/hash:71
  this[kHandle] = new _Hash(algorithm, xofLen);
                  ^

Error: error:0308010C:digital envelope routines::unsupported
    at new Hash (node:internal/crypto/hash:71:19)
    at Object.createHash (node:crypto:140:10)
    at module.exports (C:\Users\*\Desktop\arcade-songs\node_modules\webpack\lib\util\createHash.js:135:53)
    at NormalModule._initBuildHash (C:\Users\*\Desktop\arcade-songs\node_modules\webpack\lib\NormalModule.js:417:16)    at handleParseError (C:\Users\*\Desktop\arcade-songs\node_modules\webpack\lib\NormalModule.js:471:10)
    at C:\Users\*\Desktop\arcade-songs\node_modules\webpack\lib\NormalModule.js:503:5
    at C:\Users\*\Desktop\arcade-songs\node_modules\webpack\lib\NormalModule.js:358:12
    at C:\Users\*\Desktop\arcade-songs\node_modules\loader-runner\lib\LoaderRunner.js:373:3
    at iterateNormalLoaders (C:\Users\*\Desktop\arcade-songs\node_modules\loader-runner\lib\LoaderRunner.js:214:10)
    at Array.<anonymous> (C:\Users\*\Desktop\arcade-songs\node_modules\loader-runner\lib\LoaderRunner.js:205:4)
    at Storage.finished (C:\Users\*\Desktop\arcade-songs\node_modules\enhanced-resolve\lib\CachedInputFileSystem.js:55:16)
    at C:\Users\*\Desktop\arcade-songs\node_modules\enhanced-resolve\lib\CachedInputFileSystem.js:91:9
    at C:\Users\*\Desktop\arcade-songs\node_modules\graceful-fs\graceful-fs.js:123:16
    at FSReqCallback.readFileAfterClose [as oncomplete] (node:internal/fs/read_file_context:68:3) {
  opensslErrorStack: [ 'error:03000086:digital envelope routines::initialization error' ],
  library: 'digital envelope routines',
  reason: 'unsupported',
  code: 'ERR_OSSL_EVP_UNSUPPORTED'
}

Node.js v19.4.0
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
```
To resolve the above error, lowering the NodeJS version to 16 was the best solution for me.

It would be better to fix the error or explicitly indicate the NodeJS version available.

**Comment by zetaraku** [comment]
Thanks for your reporting!
I'll investigate this problem later. :)

**Comment by zetaraku** [comment]
Hi @byongshintv! I found this is an open issue at https://github.com/nuxt/nuxt.js/issues/10844.

Quoted from https://github.com/nuxt/nuxt.js/issues/10844#issuecomment-1336275502:

> To be clear, the reason you're not able to build is that webpack 4, which is used by Nuxt 2, depends on that algorithm for hashing build outputs. But that is a build-time dependency, not a runtime dependency. Nor is it a security risk at build time, as what is being hashed is just the content of the generated files.

You can fix this problem by setting `--openssl-legacy-provider` option in `NODE_OPTIONS` or downgrading Node.js to `16.x`.

I created a PR to update the prerequisites: #10
Please take a look and see if there's anything that can be improved. :)

**Comment by zetaraku** [comment]
It seems that setting the `--openssl-legacy-provider` option globally is not a good idea and doesn't work on Netlify.
I'll document the version requirement for now and wait for Nuxt to resolve this issue.

**Comment by byongshintv** [comment]
Thanks for your efforts to resolve this issue!

**Comment by zetaraku** [comment]
Hi @byongshintv!

The issue from Nuxt is resolved in v2.16.2 and the dependencies of this project have been updated.
Now the project can be run even on Node.js v20.0.0!
